### DISCUSSION TITLE
Pushkin Prep Issue - 2.2.10

### DISCUSSION THREAD

**Discussion by LClarkeW** [l3atbc-datadog/pushkin#183] [Q&A]
Hi there! I'm not sure how active the discussion forum is but I figured I'd try and ask!

I'm running through the quick start guide using the most recent version of pushkin and have run into an issue. After running `pushkin prep`, I get output consistent to what should be expected, but the command never naturally terminates, just continues to print ellipsis. Not sure if it's relevant, but the last non-filler lines read: `Finished building all experiment workers/
updating docker-compose.dev.yml/ Writing out front-end config`. 

Looking through the respective directories, it seems like the command is doing it's job. When manually exiting the command, I am still able to successfully run `pushkin start`, but the pushkin server closes seconds after deploying. I'm assuming the two issues are related. 

I could be missing something obvious but in trying to sort out exactly what it is, I seem to have come up against a wall.
Any help is appreciated, thanks!

**Comment by jkhartshorne** [comment]
We've been in the process of rebuilding the CLI. A number of things stopped working, and it turns out that migrating to newer versions of packages involves a lot of breaking changes. If you monitor "Issues", you'll see a lot of activity right now.
